### PR TITLE
feat(Tetra Legends): add presence

### PR DESCRIPTION
**Pull request by DXL44**
## Description 
Adds presence for [Tetra Legends](https://tetralegends.app/), a single player browser-based Tetris clone. Displays the current mode you're playing or if you're in the menus, as well as your total play time.
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<a href="https://imgur.com/9GCk8Wt"></a>
<a href="https://imgur.com/1Fjb6LJ"></a>
<a href="https://imgur.com/Up4P738"></a>
</details>

### CODE REVIEW

File: websites/T/Tetra Legends/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+		"name": "dxl44"
+	},
+	"service": "Tetra Legends",
+	"description": {
+		"en": "A browser-based clone of Tetris with both new and classic gamemodes."
+	},
+	"url": "tetralegends.app",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/9c3lJCA.png",
+	"thumbnail": "https://i.imgur.com/Su9t23q.png",
```
Comment (Bas950) [inline websites/T/Tetra Legends/metadata.json:14]: seems like a few pixels of something else is showing at the bottom

**Review by Bas950 — COMMENTED** [review]
Can you add more data? 
Score, level, etc.? Maybe as a slideshow?

File: websites/T/Tetra Legends/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+		"id": "300707889717444608",
+		"name": "dxl44"
+	},
+	"service": "Tetra Legends",
+	"description": {
+		"en": "A browser-based clone of Tetris with both new and classic gamemodes."
+	},
+	"url": "tetralegends.app",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/9c3lJCA.png",
```
Comment (theusaf) [inline websites/T/Tetra Legends/metadata.json:13]: This logo may be hard to see for light-mode users.

File: websites/T/Tetra Legends/presence.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	gameTime = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/UV7X7U4.jpg",
+		details: "In Menus",
+		startTimestamp: gameTime,
+	};
+
+	if (document.querySelector("#game-container").classList[0] === "hidden")
```
Comment (theusaf) [inline websites/T/Tetra Legends/presence.ts:13]: Alternatively:
```suggestion
	if (document.querySelector("#game-container.hidden"))
```

File: websites/T/Tetra Legends/presence.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+		presenceData.details = "In Menus";
+	else if (document.querySelector("header.header")) {
+		presenceData.details = `Mode: ${
+			document.querySelector("header.header").innerHTML
```
Comment (theusaf) [inline websites/T/Tetra Legends/presence.ts:17]: Prefer `.textContent` over `.innerHTML`
```suggestion
			document.querySelector("header.header").textContent
```

**Comment by Bas950** [comment]
@DXL44 Any updates?

**Comment by DXL44** [comment]
Apologies, I forgot about this. I'll go ahead and close it as I don't have interest in finishing it anymore

**Pull request closed without merging**